src/test/unit/DelayedWithdrawalRouterUnit.t.sol

```diff
@@ -131,7 +143,13 @@ contract DelayedWithdrawalRouterUnitTests is Test {
         delayedWithdrawalRouter.createDelayedWithdrawal{value: delayedWithdrawalAmount}(podOwner, address(0));
     }
 
-    function testClaimDelayedWithdrawals(uint8 delayedWithdrawalsToCreate, uint8 maxNumberOfDelayedWithdrawalsToClaim, uint256 pseudorandomNumber_, address recipient, bool useOverloadedFunction)
+    function testClaimDelayedWithdrawals(
+        uint8 delayedWithdrawalsToCreate,
+        uint8 maxNumberOfDelayedWithdrawalsToClaim,
+        uint256 pseudorandomNumber_,
```

Comment:
looked into the file and doesn't seem like we use this param?

Reply:
looks like we use the _getPseudorandomNumber function though

Reply:
we are using indeed using this input. we set the storage `_pseudorandomNumber` equal to the `pseudorandomNumber_`. The storage value is then used inside the `_getPseudorandomNumber` function.